from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Admin_creating_user(object):
    def setupUi(self, Admin_creating_user):
        Admin_creating_user.setObjectName("Admin_creating_user")
        Admin_creating_user.resize(633, 466)
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(14)
        Admin_creating_user.setFont(font)
        Admin_creating_user.setWindowTitle("Creating User")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("Myicon.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        Admin_creating_user.setWindowIcon(icon)
        Admin_creating_user.setStyleSheet("background-color:#c5d5cb;")
        Admin_creating_user.setIconSize(QtCore.QSize(57, 29))
        Admin_creating_user.setTabShape(QtWidgets.QTabWidget.Triangular)
        self.centralwidget = QtWidgets.QWidget(Admin_creating_user)
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(14)
        self.centralwidget.setFont(font)
        self.centralwidget.setObjectName("centralwidget")
        self.openGLWidget = QtWidgets.QOpenGLWidget(self.centralwidget)
        self.openGLWidget.setGeometry(QtCore.QRect(-300, 330, 300, 200))
        self.openGLWidget.setObjectName("openGLWidget")
        self.logoutButton = QtWidgets.QCommandLinkButton(self.centralwidget)
        self.logoutButton.setGeometry(QtCore.QRect(530, 0, 111, 41))
        font = QtGui.QFont()
        font.setFamily("Segoe UI")
        font.setPointSize(11)
        self.logoutButton.setFont(font)
        self.logoutButton.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("logout.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.logoutButton.setIcon(icon1)
        self.logoutButton.setIconSize(QtCore.QSize(24, 24))
        self.logoutButton.setObjectName("logoutButton")
        self.back = QtWidgets.QCommandLinkButton(self.centralwidget)
        self.back.setGeometry(QtCore.QRect(0, -10, 61, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI")
        font.setPointSize(14)
        self.back.setFont(font)
        self.back.setText("")
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("icons8-left-arrow-64.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.back.setIcon(icon2)
        self.back.setIconSize(QtCore.QSize(40, 40))
        self.back.setObjectName("back")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(110, 70, 131, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(110, 200, 91, 21))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(110, 130, 171, 41))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.full_name = QtWidgets.QLineEdit(self.centralwidget)
        self.full_name.setGeometry(QtCore.QRect(290, 70, 201, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.full_name.setFont(font)
        self.full_name.setText("")
        self.full_name.setObjectName("full_name")
        self.father_name = QtWidgets.QLineEdit(self.centralwidget)
        self.father_name.setGeometry(QtCore.QRect(290, 130, 201, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.father_name.setFont(font)
        self.father_name.setObjectName("father_name")
        self.voter_id = QtWidgets.QLineEdit(self.centralwidget)
        self.voter_id.setGeometry(QtCore.QRect(290, 190, 131, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.voter_id.setFont(font)
        self.voter_id.setObjectName("voter_id")
        self.date_of_birth = QtWidgets.QDateEdit(self.centralwidget)
        self.date_of_birth.setGeometry(QtCore.QRect(290, 260, 121, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(11)
        self.date_of_birth.setFont(font)
        self.date_of_birth.setObjectName("date_of_birth")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(110, 260, 141, 31))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton.setGeometry(QtCore.QRect(480, 360, 91, 41))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_2.setGeometry(QtCore.QRect(320, 360, 91, 41))
        font = QtGui.QFont()
        font.setFamily("Comic Sans MS")
        font.setPointSize(12)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        Admin_creating_user.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(Admin_creating_user)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 633, 21))
        self.menubar.setObjectName("menubar")
        Admin_creating_user.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(Admin_creating_user)
        self.statusbar.setObjectName("statusbar")
        Admin_creating_user.setStatusBar(self.statusbar)

        self.retranslateUi(Admin_creating_user)
        QtCore.QMetaObject.connectSlotsByName(Admin_creating_user)

    def retranslateUi(self, Admin_creating_user):
        _translate = QtCore.QCoreApplication.translate
        self.logoutButton.setText(_translate("Admin_creating_user", "Logout"))
        self.label.setText(_translate("Admin_creating_user", "Full Name"))
        self.label_2.setText(_translate("Admin_creating_user", "Voter ID"))
        self.label_3.setText(_translate("Admin_creating_user", "Father Name"))
        self.label_4.setText(_translate("Admin_creating_user", "Date of Birth"))
        self.pushButton.setText(_translate("Admin_creating_user", "Create"))
        self.pushButton_2.setText(_translate("Admin_creating_user", "Cancel"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Admin_creating_user = QtWidgets.QMainWindow()
    ui = Ui_Admin_creating_user()
    ui.setupUi(Admin_creating_user)
    Admin_creating_user.show()
    sys.exit(app.exec_())
